lib/fortnox/api/validators/edi_information.rb

```diff
@@ -0,0 +1,16 @@
+require "fortnox/api/validators/base"
+
+module Fortnox
+  module API
+    module Validator
+      class EDIInformation
+
+        extend Fortnox::API::Validator::Base
+
+        using_validations do
+          # No validations needed, in other words always valid.
```

Comment:
Adding a spec to make sure that the two classes respond to the same messages is enough. So no shared superclass, think duck typing, not inheritance.

Just verifying that the parts of the API that we care about matches between the two implementations is the right way to go here. It's like contracts in Go or interfaces in Java, but without runtime environment enforcement.